types/trouter/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for trouter 2.0
+// Project: https://github.com/lukeed/trouter
+// Definitions by: Markus Lanz <https://github.com/stahlstift>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare namespace Trouter {
+    interface FindResult<T> {
+        params: { [k: string]: string; };
+        handlers: T[];
+    }
+
+    type ValidMethods = 'get' | 'head' | 'patch' | 'options' |
```

Comment:
Is there anyway we can import/use the [`http.METHODS`](https://nodejs.org/api/http.html#http_http_methods) list? They're all valid – Trouter only attaches these to the prototype for convenience as they're the more common methods.

Also, if we need/are to use an embedded list, these should be capitalized and `"*"` is also used for `.all()` ... but that's changing in next version so can be omitted.

Reply:
http.METHODS is declared as
```ts
// @types/node": "^11.10.4"     
// http.d.ts:243    
const METHODS: string[];
``` 
so - no it's not possible .

I will change the methods to uppercase and ignoring like you suggest "*"